diff -urN a/src/terminal-window.c b/src/terminal-window.c
--- a/src/terminal-window.c	2021-08-05 09:25:06.000000000 -0700
+++ b/src/terminal-window.c	2022-06-12 11:05:09.703101244 -0700
@@ -1887,12 +1887,12 @@
 
         /* File menu */
         {
-            "FileNewWindow", "utilities-terminal", N_("Open _Terminal"), "<shift><control>N",
+            "FileNewWindow", "utilities-terminal", N_("Open _Terminal"), "<super>N",
             NULL,
             G_CALLBACK (file_new_window_callback)
         },
         {
-            "FileNewTab", STOCK_NEW_TAB, N_("Open Ta_b"), "<shift><control>T",
+            "FileNewTab", STOCK_NEW_TAB, N_("Open Ta_b"), "<super>T",
             NULL,
             G_CALLBACK (file_new_tab_callback)
         },
@@ -1907,24 +1907,24 @@
             G_CALLBACK (file_save_contents_callback)
         },
         {
-            "FileCloseTab", "window-close", N_("C_lose Tab"), "<shift><control>W",
+            "FileCloseTab", "window-close", N_("C_lose Tab"), "<super>W",
             NULL,
             G_CALLBACK (file_close_tab_callback)
         },
         {
-            "FileCloseWindow", "window-close", N_("_Close Window"), "<shift><control>Q",
+            "FileCloseWindow", "window-close", N_("_Close Window"), "<super>Q",
             NULL,
             G_CALLBACK (file_close_window_callback)
         },
 
         /* Edit menu */
         {
-            "EditCopy", "edit-copy", N_("_Copy"), "<shift><control>C",
+            "EditCopy", "edit-copy", N_("_Copy"), "<super>C",
             NULL,
             G_CALLBACK (edit_copy_callback)
         },
         {
-            "EditPaste", "edit-paste", N_("_Paste"), "<shift><control>V",
+            "EditPaste", "edit-paste", N_("_Paste"), "<super>V",
             NULL,
             G_CALLBACK (edit_paste_callback)
         },
@@ -1934,7 +1934,7 @@
             G_CALLBACK (edit_paste_callback)
         },
         {
-            "EditSelectAll", "edit-select-all", N_("Select _All"), "<shift><control>A",
+            "EditSelectAll", "edit-select-all", N_("Select _All"), "<super>A",
             NULL,
             G_CALLBACK (edit_select_all_callback)
         },
@@ -1956,50 +1956,50 @@
 
         /* View menu */
         {
-            "ViewZoomIn", "zoom-in", N_("Zoom _In"), "<control>plus",
+            "ViewZoomIn", "zoom-in", N_("Zoom _In"), "<super>plus",
             NULL,
             G_CALLBACK (view_zoom_in_callback)
         },
         {
-            "ViewZoomOut", "zoom-out", N_("Zoom _Out"), "<control>minus",
+            "ViewZoomOut", "zoom-out", N_("Zoom _Out"), "<super>minus",
             NULL,
             G_CALLBACK (view_zoom_out_callback)
         },
         {
-            "ViewZoom100", "zoom-original", N_("_Normal Size"), "<control>0",
+            "ViewZoom100", "zoom-original", N_("_Normal Size"), "<super>0",
             NULL,
             G_CALLBACK (view_zoom_normal_callback)
         },
 
         /* Search menu */
         {
-            "SearchFind", "edit-find", N_("_Find..."), "<shift><control>F",
+            "SearchFind", "edit-find", N_("_Find..."), "<super>F",
             NULL,
             G_CALLBACK (search_find_callback)
         },
         {
-            "SearchFindNext", NULL, N_("Find Ne_xt"), "<shift><control>H",
+            "SearchFindNext", NULL, N_("Find Ne_xt"), "<super>H",
             NULL,
             G_CALLBACK (search_find_next_callback)
         },
         {
-            "SearchFindPrevious", NULL, N_("Find Pre_vious"), "<shift><control>G",
+            "SearchFindPrevious", NULL, N_("Find Pre_vious"), "<super>G",
             NULL,
             G_CALLBACK (search_find_prev_callback)
         },
         {
-            "SearchClearHighlight", NULL, N_("_Clear Highlight"), "<shift><control>J",
+            "SearchClearHighlight", NULL, N_("_Clear Highlight"), "<super>J",
             NULL,
             G_CALLBACK (search_clear_highlight_callback)
         },
 #if 0
         {
-            "SearchGoToLine", "go-jump", N_("Go to _Line..."), "<shift><control>I",
+            "SearchGoToLine", "go-jump", N_("Go to _Line..."), "<super>I",
             NULL,
             G_CALLBACK (search_goto_line_callback)
         },
         {
-            "SearchIncrementalSearch", "edit-find", N_("_Incremental Search..."), "<shift><control>K",
+            "SearchIncrementalSearch", "edit-find", N_("_Incremental Search..."), "<super>K",
             NULL,
             G_CALLBACK (search_incremental_search_callback)
         },
@@ -2043,22 +2043,22 @@
 
         /* Tabs menu */
         {
-            "TabsPrevious", NULL, N_("_Previous Tab"), "<control>Page_Up",
+            "TabsPrevious", NULL, N_("_Previous Tab"), "<super>Page_Up",
             NULL,
             G_CALLBACK (tabs_next_or_previous_tab_cb)
         },
         {
-            "TabsNext", NULL, N_("_Next Tab"), "<control>Page_Down",
+            "TabsNext", NULL, N_("_Next Tab"), "<super>Page_Down",
             NULL,
             G_CALLBACK (tabs_next_or_previous_tab_cb)
         },
         {
-            "TabsMoveLeft", NULL, N_("Move Tab _Left"), "<shift><control>Page_Up",
+            "TabsMoveLeft", NULL, N_("Move Tab _Left"), "<sshift><super>Page_Up",
             NULL,
             G_CALLBACK (tabs_move_left_callback)
         },
         {
-            "TabsMoveRight", NULL, N_("Move Tab _Right"), "<shift><control>Page_Down",
+            "TabsMoveRight", NULL, N_("Move Tab _Right"), "<shift><super>Page_Down",
             NULL,
             G_CALLBACK (tabs_move_right_callback)
         },
