diff -urN a/src/eggtreemultidnd.c b/src/eggtreemultidnd.c
--- a/src/eggtreemultidnd.c	2021-08-06 16:13:09.000000000 -0700
+++ b/src/eggtreemultidnd.c	2022-05-31 08:48:06.880662174 -0700
@@ -417,7 +417,7 @@
 
   if (path)
     {
-      gboolean call_parent = (event->state & (GDK_CONTROL_MASK | GDK_SHIFT_MASK) ||
+      gboolean call_parent = (event->state & (GDK_SUPER_MASK | GDK_SHIFT_MASK) ||
 			      !gtk_tree_selection_path_is_selected (selection, path) ||
 			      event->button != 1);
 
diff -urN a/src/fr-window.c b/src/fr-window.c
--- a/src/fr-window.c	2021-08-06 16:13:09.000000000 -0700
+++ b/src/fr-window.c	2022-05-31 08:48:06.883995160 -0700
@@ -3878,7 +3878,7 @@
 
 			if ((gtk_tree_path_compare (window->priv->path_clicked, path) == 0)
 			    && window->priv->single_click
-			    && ! ((event->state & GDK_CONTROL_MASK) || (event->state & GDK_SHIFT_MASK))) {
+			    && ! ((event->state & GDK_SUPER_MASK) || (event->state & GDK_SHIFT_MASK))) {
 				gtk_tree_view_set_cursor (GTK_TREE_VIEW (widget),
 							  path,
 							  NULL,
@@ -5913,7 +5913,7 @@
 
 	/* Add a hidden short cut Ctrl-Q for power users */
 	gtk_accel_group_connect (gtk_ui_manager_get_accel_group (ui),
-				 GDK_KEY_q, GDK_CONTROL_MASK, 0,
+				 GDK_KEY_q, GDK_SUPER_MASK, 0,
 				 g_cclosure_new_swap (G_CALLBACK (fr_window_close), window, NULL));
 
 
diff -urN a/src/ui.h b/src/ui.h
--- a/src/ui.h	2021-08-06 16:13:09.000000000 -0700
+++ b/src/ui.h	2022-05-31 08:48:24.828794853 -0700
@@ -55,28 +55,28 @@
 	  N_("Add a folder to the archive"),
 	  G_CALLBACK (activate_action_add_folder) },
 	{ "Close", "window-close",
-	  N_("_Close"), "<control>W",
+	  N_("_Close"), "<super>W",
 	  N_("Close the current archive"),
 	  G_CALLBACK (activate_action_close) },
 	{ "Contents", "help-browser",
-	  N_("Contents"), "F1",
+	  N_("Contents"), NULL,
 	  N_("Display the Engrampa Manual"),
 	  G_CALLBACK (activate_action_manual) },
 
 	{ "Copy", "edit-copy",
-	  N_("_Copy"), "<control>C",
+	  N_("_Copy"), "<super>C",
 	  N_("Copy the selection"),
 	  G_CALLBACK (activate_action_copy) },
 	{ "Cut", "edit-cut",
-	  N_("Cu_t"), "<control>X",
+	  N_("Cu_t"), "<super>X",
 	  N_("Cut the selection"),
 	  G_CALLBACK (activate_action_cut) },
 	{ "Paste", "edit-paste",
-	  N_("_Paste"), "<control>V",
+	  N_("_Paste"), "<super>V",
 	  N_("Paste the clipboard"),
 	  G_CALLBACK (activate_action_paste) },
 	{ "Rename", NULL,
-	  N_("_Rename…"), "F2",
+	  N_("_Rename…"), "<super>E",
 	  N_("Rename the selection"),
 	  G_CALLBACK (activate_action_rename) },
 	{ "Delete", "edit-delete",
@@ -85,15 +85,15 @@
 	  G_CALLBACK (activate_action_delete) },
 
 	{ "CopyFolderFromSidebar", "edit-copy",
-	  N_("_Copy"), "<control>C",
+	  N_("_Copy"), "<super>C",
 	  N_("Copy the selection"),
 	  G_CALLBACK (activate_action_copy_folder_from_sidebar) },
 	{ "CutFolderFromSidebar", "edit-cut",
-	  N_("Cu_t"), "<control>X",
+	  N_("Cu_t"), "<super>X",
 	  N_("Cut the selection"),
 	  G_CALLBACK (activate_action_cut_folder_from_sidebar) },
 	{ "PasteFolderFromSidebar", "edit-paste",
-	  N_("_Paste"), "<control>V",
+	  N_("_Paste"), "<super>V",
 	  N_("Paste the clipboard"),
 	  G_CALLBACK (activate_action_paste_folder_from_sidebar) },
 	{ "RenameFolderFromSidebar", NULL,
@@ -106,11 +106,11 @@
 	  G_CALLBACK (activate_action_delete_folder_from_sidebar) },
 
 	{ "DeselectAll", NULL,
-	  N_("Dese_lect All"), "<shift><control>A",
+	  N_("Dese_lect All"), "<shift><super>A",
 	  N_("Deselect all files"),
 	  G_CALLBACK (activate_action_deselect_all) },
 	{ "Extract", "extract-archive",
-	  N_("_Extract…"), "<control>E",
+	  N_("_Extract…"), "<shift><super>E",
 	  N_("Extract files from the archive"),
 	  G_CALLBACK (activate_action_extract) },
 	{ "ExtractFolderFromSidebar", "extract-archive",
@@ -122,7 +122,7 @@
 	  N_("Extract files from the archive"),
 	  G_CALLBACK (activate_action_extract) },
 	{ "Find", "edit-find",
-	  N_("Find…"), "<control>F",
+	  N_("Find…"), "<super>F",
 	  NULL,
 	  G_CALLBACK (activate_action_find) },
 
@@ -131,15 +131,15 @@
 	  N_("View the output produced by the last executed command"),
 	  G_CALLBACK (activate_action_last_output) },
 	{ "New", "document-new",
-	  N_("New…"), "<control>N",
+	  N_("New…"), "<super>N",
 	  N_("Create a new archive"),
 	  G_CALLBACK (activate_action_new) },
 	{ "Open", "document-open",
-	  N_("Open…"), "<control>O",
+	  N_("Open…"), "<super>O",
 	  N_("Open archive"),
 	  G_CALLBACK (activate_action_open) },
 	{ "Open_Toolbar", "document-open",
-	  N_("_Open"), "<control>O",
+	  N_("_Open"), "<super>O",
 	  N_("Open archive"),
 	  G_CALLBACK (activate_action_open) },
 	{ "OpenSelection", NULL,
@@ -151,23 +151,23 @@
 	  N_("Specify a password for this archive"),
 	  G_CALLBACK (activate_action_password) },
 	{ "Properties", "document-properties",
-	  N_("_Properties"), "<alt>Return",
+	  N_("_Properties"), "<super>I",
 	  N_("Show archive properties"),
 	  G_CALLBACK (activate_action_properties) },
 	{ "Reload", "view-refresh",
-	  N_("_Refresh"), "<control>R",
+	  N_("_Refresh"), "<super>R",
 	  N_("Reload current archive"),
 	  G_CALLBACK (activate_action_reload) },
 	{ "SaveAs", "document-save-as",
-	  N_("Save As…"), NULL,
+	  N_("Save As…"), "<shift><super>S",
 	  N_("Save the current archive with a different name"),
 	  G_CALLBACK (activate_action_save_as) },
 	{ "SelectAll", "edit-select-all",
-	  N_("Select _All"), "<control>A",
+	  N_("Select _All"), "<super>A",
 	  N_("Select all files"),
 	  G_CALLBACK (activate_action_select_all) },
 	{ "Stop", "process-stop",
-	  N_("_Stop"), "Escape",
+	  N_("_Stop"), "<super>.",
 	  N_("Stop current operation"),
 	  G_CALLBACK (activate_action_stop) },
 	{ "TestArchive", NULL,
@@ -175,19 +175,19 @@
 	  N_("Test whether the archive contains errors"),
 	  G_CALLBACK (activate_action_test_archive) },
 	{ "ViewSelection", "document-open",
-	  N_("_Open"), "<control>O",
+	  N_("_Open"), "<super>O",
 	  N_("Open the selected file"),
 	  G_CALLBACK (activate_action_view_or_open) },
 	{ "ViewSelection_Toolbar", "document-open",
-	  N_("_Open"), "<control>O",
+	  N_("_Open"), "<super>O",
 	  N_("Open the selected file"),
 	  G_CALLBACK (activate_action_view_or_open) },
 	{ "OpenFolder", "document-open",
-	  N_("_Open"), "<control>O",
+	  N_("_Open"), "<super>O",
 	  N_("Open the selected folder"),
 	  G_CALLBACK (activate_action_open_folder) },
 	{ "OpenFolderFromSidebar", "document-open",
-	  N_("_Open"), "<control>O",
+	  N_("_Open"), "<super>O",
 	  N_("Open the selected folder"),
 	  G_CALLBACK (activate_action_open_folder_from_sidebar) },
 
@@ -229,7 +229,7 @@
 	  G_CALLBACK (activate_action_sort_reverse_order),
 	  FALSE },
 	{ "ViewFolders", NULL,
-	  N_("_Folders"), "F9",
+	  N_("_Folders"), "<shift><super>T",
 	  N_("View the folders pane"),
 	  G_CALLBACK (activate_action_view_folders),
 	  FALSE },
@@ -239,10 +239,10 @@
 
 static GtkRadioActionEntry view_as_entries[] = {
 	{ "ViewAllFiles", NULL,
-	  N_("View All _Files"), "<control>1",
+	  N_("View All _Files"), "<super>1",
 	  " ", FR_WINDOW_LIST_MODE_FLAT },
 	{ "ViewAsFolder", NULL,
-	  N_("View as a F_older"), "<control>2",
+	  N_("View as a F_older"), "<super>2",
 	  " ", FR_WINDOW_LIST_MODE_AS_DIR },
 };
 static guint n_view_as_entries = G_N_ELEMENTS (view_as_entries);
