--- a/src/shell/e-shell-window.c.orig	2018-09-22 12:11:06.901023138 +0900
+++ b/src/shell/e-shell-window.c	2018-09-22 12:14:40.137685266 +0900
@@ -67,29 +67,29 @@
 
 static const char *css =
 ".table-header {\
-	border-bottom: 1px solid @borders;\
+	border-bottom: 1px solid #b0b0b0;\
 }\
 .button {\
 	padding: 3px 5px;\
 }\
 .table-header .button {\
-	border-right: 1px solid @borders;\
+	border-right: 1px solid #b0b0b0;\
 }\
 .table-header .button.last {\
 	border-right: none;\
 }\
 toolbar {\
-	border-bottom: 1px solid @borders;\
+	border-bottom: 1px solid #b0b0b0;\
 }\
 .taskbar border {\
 	border-width: 1px 0 0 0;\
 }\
 .header-box {\
-	border-bottom: 1px solid @borders;\
+	border-bottom: 1px solid #b0b0b0;\
 	padding: 3px;\
 }\
 #e-attachment-bar {\
-	border-top: 1px solid @borders;\
+	border-top: 1px solid #b0b0b0;\
 }\
 ";
 
